README.md

```diff
@@ -1,26 +1,31 @@
-# log_parser
+# Spark Log Parser
 The **Parser for Apache Spark** parses unmodified Apache Spark History Server Event logs.
 
 Parsed logs contain metadata pertaining to your Apache Spark application execution. Particularly, the runtime for a task, the amount of data read & written, the amount of memory used, etc. These logs do not contain
-sensitive information such as the data that your Apache Spark application is processing. Below is an example of the output of the log parser
+sensitive information such as the data that your Apache Spark application is processing. Below is an example of the output of the log parser.
 ![Output of Log Parser](docs/output.png)
 
 # Installation
-Clone this repo to the desired directory.
+Install the package in this repo to your Python 3 environment, e.g.
+```shell
+pip3 install https://github.com/synccomputingcode/spark_log_parser/archive/v0.0.1.tar.gz
```

Comment:
If the version changes then this will have to be updated. Is it possible to write the install so that it automatically downloads the most recent version?

Reply:
This is relevant to the pip-install in sync_backend too. It would be ideal if we didn't have to change the version number in the install every time a commit is made to the spark_log_parser repo.

Reply:
Yeah, users can do this to always get the latest:
```
pip3 install https://github.com/synccomputingcode/spark_log_parser/archive/main.tar.gz
```